web/src/resize.ts

```diff
@@ -17,14 +17,15 @@ function get_new_heights(): {
     buddy_list_wrapper_max_height: number;
 } {
     const viewport_height = message_viewport.height();
+    // Add some gap for bottom element to be properly visible.
+    const GAP = 15;
```

Comment:
@karlstolley FYI -- we will probably want to figure out a better value to use here... but this may not be worth much attention, since ideally we want to use pure CSS solution that lets us delete `stream_filters_max_height`.